tests/Php54/Php54Test.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php54;
+
+class Php54Test extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @dataProvider provideClassUsesValid
+     */
+    public function testClassUsesValid($classOrObject)
+    {
+        $this->assertSame(array(), class_uses($classOrObject));
+    }
+
+    public function provideClassUsesValid()
+    {
+        return array(
+            array('stdClass'),
+            array(new \stdClass()),
+            array('Iterator'),
+        );
+    }
+
+    public function testClassUsesInvalid()
+    {
+        $this->assertFalse(@class_uses('NotDefined'));
```

Comment:
How can you polyfill a warning trigger? And I was unsure how to test triggered errors in PHPunit, so I skipped that test.